platform/android/java/app/build.gradle

```diff
@@ -45,6 +45,7 @@ dependencies {
         // Custom build mode. In this scenario this project is the only one around and the Godot
         // library is available through the pre-generated godot-lib.*.aar android archive files.
         debugImplementation fileTree(dir: 'libs/debug', include: ['*.jar', '*.aar'])
+        releaseDebugmIplementation fileTree(dir: 'libs/debug', include: ['*.jar', '*.aar'])
```

Comment:
Typo!

Is this needed?

Reply:
Hmm.. interesting, Im not sure how release_debug works 😅

Reply:
Addressed in https://github.com/thebestnom/godot/pull/2